yarn.lock

```diff
@@ -0,0 +1,97 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file should be removed; yarn doesn't work on older nodes, and it will break travis-ci to have this file present in the repo.